src/leiningen/new/re_frame/project.clj

```diff
@@ -1,19 +1,19 @@
 (defproject {{ns-name}} "0.1.0-SNAPSHOT"
-  :dependencies [[org.clojure/clojure "1.8.0"]
-                 [org.clojure/clojurescript "1.9.908"]
+  :dependencies [[org.clojure/clojure "1.9.0"]
```

Comment:
Can you leave this at clojure 1.8.0?  

Bumping to 1.9.0 will break figwheel for users with java 8 on their computer. For the time being, I am reluctant to require people to upgrade their java version.

Reply:
will do